### ISSUE TITLE
The bower version converter has stopped working 

### ISSUE THREAD

**Issue by karlbennett** [webjars/webjars#1636]
Looks like the bower version converter isn't working any more. Noticed this with the version 2.0.x releases of the [Polymer bower webjar](https://search.maven.org/#search%7Cgav%7C1%7Cg%3A%22org.webjars.bower%22%20AND%20a%3A%22polymer%22).
Version 1.8.0
```xml
<dependencies>
         
        <dependency>
            <groupId>org.webjars.bower</groupId>
            <artifactId>webcomponentsjs</artifactId>
            <version>[0.7.24,0.8)</version>
        </dependency>
         
    </dependencies>
``` 
Version 2.0.0
```xml
<dependencies>
         
        <dependency>
            <groupId>org.webjars.bower</groupId>
            <artifactId>github-com-webcomponents-shadycss</artifactId>
            <version>^v1.0.0</version>
        </dependency>
         
        <dependency>
            <groupId>org.webjars.bower</groupId>
            <artifactId>webcomponentsjs</artifactId>
            <version>^v1.0.0</version>
        </dependency>
         
         
    </dependencies>
```
As you can see the dependencies now contain npm style version ranges. It looks like it could be because the version numbers start with a 'v'.

**Comment by jamesward** [comment]
Thanks for reporting.  Here is the original `bower.json`: https://github.com/Polymer/polymer/blob/v2.0.2/bower.json

**Comment by karlbennett** [comment]
Sent pull request for this #1637.

**Comment by Legioth** [comment]
As already noted, this seems to happen only when there's a `v` prefix in the version number, which is a bug in itself (see https://github.com/webjars/webjars/issues/1656).

As an example, https://github.com/vaadin/vaadin-grid/blob/v4.0.0-alpha5/bower.json has dependencies for `"polymer": "^2.0.0"` (no `v` prefix) and `"iron-a11y-keys-behavior": "^v2.0.0"` (with a `v` prefix). The resulting `pom.xml` has a proper version range for `polymer`: `[2.0.0,3)`, but a broken one for `iron-a11y-keys-behavior`: `^v2.0.0`.

Based on this observation, I would suggest not doing anything about this issue but instead fix
https://github.com/webjars/webjars/issues/1656 which causes these invalid version numbers to be used in the first place.

**Comment by jamesward** [comment]
Fixed via #1702
